packages/frameworks/frameworks.json

```diff
@@ -174,6 +262,14 @@
           "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"sirv-cli\":\\s*\".+?\"[^}]*}"
         }
       ]
+    },
+    "settings": {
+      "buildCommand": {
+        "placeholder": "`polymer build` or `build` from `package.json`"
```

Comment:
Not sure if Polymer again here makes sense? It's already being defined above.

Reply:
Copy-Paste mistake, fixed it